oshi-core/src/main/java/oshi/hardware/platform/windows/WindowsCentralProcessor.java

```diff
@@ -79,6 +79,28 @@
     private static final String PROCESSOR = "Processor";
 
     private Map<String, Integer> numaNodeProcToLogicalProcMap;
+    
+    private static final long BOOTTIME;
+    static {
+    	boolean found = false;
+        EventLogIterator iter = new EventLogIterator(null, "System", WinNT.EVENTLOG_BACKWARDS_READ);
```

Comment:
You need to add the include for the `EventLogIterator` and `EventLogRecord` and possibly other classes.  Your IDE should help you with these... you are using an IDE, right? :)

Reply:
@dbwiddis I made changes to address all of the concerns above. I am using Eclipse to edit my code but for some reason that I don't understand fully the IDE is not picking up on the missing imports... from what I've been reading online it seems like Eclipse not suggesting including missing imports could be related to the fact the project is built with maven and the IDE isn't recognizing the project as a Java project. But I'm really unsure.

Either way, I added the imports and I have pushed those changes to this PR (although doing so appears to have closed this PR). Should I open another PR or can you access the updated code from your end? Thank you